galleries/plot_types/stats/README.rst

```diff
@@ -1,6 +1,6 @@
 .. _stats_plots:
 
-Statistics plots
-----------------
+Statistical Plots
```

Comment:
```suggestion
Statistical plots
```
? (Not sure what the rest looks like as I am on the phone, but most other headings in this PR are not in title case.)